src/components/Buttons/Button.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
```

Comment:
Can you please add a header with the description the author an the date of creation of this component please?

Reply:
ready.